src/terminal/adapter/adaptDispatch.cpp

```diff
@@ -3068,6 +3068,13 @@ bool AdaptDispatch::WindowManipulation(const DispatchTypes::WindowManipulationTy
     case DispatchTypes::WindowManipulationType::ResizeWindowInCharacters:
         _api.ResizeWindow(parameter2.value_or(0), parameter1.value_or(0));
         return true;
+    case DispatchTypes::WindowManipulationType::ReportTextSizeInCharacters:
+    {
+        const til::rect viewport = _api.GetViewport();
```

Comment:
I'm tempted to say that this is okay, but IIRC @j4james thought we should use the buffer width, not just the viewport width here.

Reply:
Adressed.